bundles/org.openhab.binding.mail/src/main/java/org/openhab/binding/mail/internal/POP3IMAPHandler.java

```diff
@@ -52,7 +52,7 @@
 
     private @NonNullByDefault({}) POP3IMAPConfig config;
     private @Nullable ScheduledFuture<?> refreshTask;
-    private String provider = "imap";
```

Comment:
Since the thingType is constant (at least in this case) we could do `private final String baseProtocol;` and set that in the constructor (like you do in the `getBaseProtocol` method. No need for a method that calls another method to get a fixed value.

The protocol can then be set in `initialize`.